### PR TITLE
Fix preconfig timing (#18)

### PR DESCRIPTION
**Pull request by nahuhh**
* Changevsync mode to lighten load on storage

db-sync-mode=fast:async:1000000 switch from "safe"
Thanks @hyc

* Bug fix

Preconfig earlier
Check for storage setup

### CODE REVIEW

File: src/install-monerod-in-termux.sh [hunk 1]
```diff
@@ -43,8 +53,6 @@ fi
 
 termux-wake-lock
 sleep 1
-pkg update -y
-pkg install nano wget termux-api jq -y
```
Comment (nahuhh) [inline src/install-monerod-in-termux.sh:47]: This needed to be executed before the prior if statement

File: src/install-monerod-in-termux.sh [hunk 2]
```diff
@@ -191,14 +199,14 @@ cd $NODE_CONFIG
...
+	#confirm-external-bind=1      # Open node (confirm). Required if binding outside of localhost
+	#restricted-rpc=1             # Prevent unsafe RPC calls.
 
 # Services
 	rpc-ssl=autodetect
   	no-zmq=1
-	no-igd=1                         # Disable UPnP port mapping
-	db-sync-mode=safe                # Slow but reliable db writes
+	no-igd=1                            # Disable UPnP port mapping
+	db-sync-mode=fast:async:1000000     # Switch to db-sync-mode=safe for slow but more reliable db writes
```
Comment (nahuhh) [inline src/install-monerod-in-termux.sh:209]: hyc's recommendation to use fast:sync instead of slow

Comment (CryptoGrampy) [reply src/install-monerod-in-termux.sh:209]: did he suggest fast:sync or fast:async

Comment (nahuhh) [reply src/install-monerod-in-termux.sh:209]: fast:async 👍
I used the parameters as advised

"completely async, never explicit flush
currently there is no db-sync-mode setting that allows that, best you can do is fast:async:1000000 (some large number of blocks between flushes)"

File: src/install-monerod-in-termux.sh [hunk 3]
```diff
@@ -23,7 +23,17 @@ esac
 
 
 # Preconfigure
+if [ -d storage/downloads ]
```
Comment (nahuhh) [inline src/install-monerod-in-termux.sh]: This line checks to see if you've already run termux-storage-setup and skips it if you have.

Seems to work as a workaround to the loss of storage access on certain devices.

**Pull request merged**